microcosm_flask/conventions/encoding.py

```diff
@@ -113,7 +116,7 @@ def remove_null_values(data):
     return data
 
 
-def dump_response_data(response_schema, response_data, status_code=200, headers=None):
+def dump_response_data(response_schema, response_data, status_code=200, headers=None, response_format=None):
```

Comment:
I think you'll make this cleaner if you abstract the format in to a `formatter` "dumps" function and have the CSV convention pass the formatter instead of text string. You can have a default formatter passed as the kwargs default or you can default to JSON if None.